.buildkite/release/config.yml

```diff
@@ -7,6 +7,7 @@ builder_queues:
   builder: builder_queue_branch
 runner_queues:
   release: release_queue_small
+  release-medium: release_queue_medium
```

Comment:
should we just call this "builder"?

Reply:
there is another `builder` on line 7, i'm also worried that people might think this belongs to the `builder_queues`, which is not since they have different set of permissions